Code/GraphMol/MolInterchange/Wrap/rdMolInterchange.cpp

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright (C) 2018 Greg Landrum
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <RDBoost/python.h>
+#include <GraphMol/GraphMol.h>
+#include <RDBoost/Wrap.h>
+
+#include <GraphMol/MolInterchange/MolInterchange.h>
+
+namespace python = boost::python;
+
+namespace {
+python::tuple JSONToMols(const std::string &jsonBlock,
```

Comment:
We don't appear to properly handle unicode yet (might need a std::wstring interface) in addition to a std::string. rapidjson appears to support this, but I don't know how it will affect the parser.

I expect this will come up at some point in the future as the python requests module tends to return unicode strings  so perhaps we should put it in as a feature request.